iceoryx_hoofs/cmake/IceoryxPackageHelper.cmake

```diff
@@ -278,22 +278,18 @@ Macro(iox_add_library)
         PUBLIC_LIBS_APPLE PRIVATE_LIBS_APPLE)
     cmake_parse_arguments(IOX "${switches}" "${arguments}" "${multiArguments}" ${ARGN} )
 
-    if ( NOT IOX_NO_PACKAGE_SETUP )
```

Comment:
This is nowhere defined and I don't know why it is present

Reply:
God knows ;)